### PR TITLE
Default Exception handler

### PR DESCRIPTION
**Pull request by elisabethnetland**
Issue #53 
Added an error controller that sends an RFC 7807-compliant payload to the client.

### CODE REVIEW

File: GiEnJul/Startup.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 
+
```
Comment (TheseusDeus) [inline GiEnJul/Startup.cs:10]: unnecessary newline

File: GiEnJul/Startup.cs [hunk 2]
```diff
@@ -53,16 +54,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
...
             app.UseRouting();
-
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllerRoute(
                     name: "default",
                     pattern: "{controller}/{action=Index}/{id?}");
             });
 
+
```
Comment (TheseusDeus) [inline GiEnJul/Startup.cs:65]: unnecessary newline

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "lockfileVersion": 1,
+  "dependencies": {
+    "problem-json": {
+      "version": "0.3.0",
+      "resolved": "https://registry.npmjs.org/problem-json/-/problem-json-0.3.0.tgz",
+      "integrity": "sha1-mLXeqE3U8ybHN96S8O52/68EdkU=",
+      "dev": true
+    }
+  }
+}
```
Comment (Sommerli) [inline package-lock.json:12]: Invalid file-placement

**Review by Sommerli — CHANGES_REQUESTED** [review]
package-lock.json should only exist in the ClientApp directory

**Pull request closed without merging**